### DISCUSSION TITLE
Where is the polaris-react components Props on the new official website ?

### DISCUSSION THREAD

**Discussion by transon-top** [Shopify/polaris-react#6466] [Q&A] [answered]
Where is the polaris-react API on the new official website ?

**Comment by heyjoethomas** [comment, marked as answer]
Hey @quanscheng the component prop tables will be coming back. We know they are important to folks. You can still find them at https://legacy.polaris.shopify.com/ in the meantime.

**Reply by transon-top** [reply]
get it . thx.  The search function in the new document is very convenient , but the component preview is expected to look like the old document (straightforward) .   : )

**Reply by alex-page** [reply]
Thanks @quanscheng totally understand the component previews need work. We are focused on bringing back the prop tables here https://github.com/Shopify/polaris/issues/6371 and updating the previews as well.
